### PR TITLE
[RIP-10] Add UT for PrintMessageSubCommand

### PR DESCRIPTION
**Pull request by linuful**
## What is the purpose of the change

1. add ut for PrintMessageSubCommand
2. adjust the code style

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: tools/src/main/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommand.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 import org.apache.rocketmq.tools.command.SubCommandException;
```
Comment (hdchen) [inline tools/src/main/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommand.java]: please  add license note

File: tools/src/test/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
+package org.apache.rocketmq.tools.command.message;
```
Comment (hdchen) [inline tools/src/test/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommandTest.java]: please add license  note!

**Comment by zongtanghu** [comment]
As above said,you should notice the apache code formate header of java class,you also should see the failed ci testing case.You can look at the logs,the logs are as follows:
[ERROR] Failed to execute goal org.apache.rat:apache-rat-plugin:0.12:check (default-cli) on project rocketmq-tools: Too many files with unapproved license: 1 See RAT report in: /home/travis/build/apache/rocketmq/tools/target/rat.txt -> [Help 1]

File: tools/src/test/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommandTest.java (hunk 2 above)
Comment (linuful) [reply tools/src/test/java/org/apache/rocketmq/tools/command/message/PrintMessageSubCommandTest.java]: ok

**Comment by linuful** [comment]
close it and renew

**Pull request closed without merging**